blog/2015/04/SCF_minutes.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: blog
+root: ../../..
+author: Karin Lagesebn
```

Comment:
@karinlag Could you check your last name? From others post is "Lagesen" but here has a extra `b`. Thanks to @arlissc to find this.